app/models/in_progress_form.rb

```diff
@@ -31,7 +31,7 @@ def self.form_for_user(form_id, user)
 
   def data_and_metadata
     {
-      form_data: JSON.parse(form_data),
```

Comment:
https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/workers/delete_attachment_job.rb#L22 needs to be changed